src/test/java/com/wikia/webdriver/testcases/discussions/ReportingPostTests.java

```diff
@@ -0,0 +1,620 @@
+    PostsListPage postsListPage = new PostsListPage().open();
+    PostsCreator postsCreator = postsListPage.getPostsCreatorMobile();
+
+    PostEntity postEntity = addPostAndGoToUserPostsPage(postsListPage, postsCreator);
+    postData.put(POST_ON_MOBILE_ON_USER_PAGE, postEntity.toData());
+
+    assertThatPostCanBeReported(postEntity);
+  }
+
+  @Test(groups = "discussions-loggedInUsersMobileReporting",
+      dependsOnMethods = {
+          "userOnMobileCanReportPostOnPostDetailsPage",
+          "anonUserOnMobileCanNotSeeReportedPostOnPostDetailsPage",
+          "userOnMobileCannotSeeReportedIndicatorOnPostsReportedByAnotherUserOnPostDetailsPageAndCanReportThatPost",
+          "moderatorOnMobileCanApproveReportedPostOnPostDetailsPage"})
+  @Execute(asUser = User.USER)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void userOnMobileCannotReReportPostOnPostDetailsPage() {
+    final String postId = postData.get(POST_ON_MOBILE_ON_POST_DETAILS).getId();
+
+    PostDetailsPage page = new PostDetailsPage().open(postId);
+
+    PostEntity postEntity = page.getPost().findPostById(postId);
+
+    assertThatPostIsNotReported(postEntity);
+    assertThatReportPostOptionIsNotAvailableFor(postEntity);
+  }
+
+  // Second user on mobile
+
+  @Test(groups = "discussions-loggedInUsersMobileReporting",
+      dependsOnMethods = "userOnMobileCanReportPostOnPostsListPage")
+  @Execute(asUser = User.USER_2)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void userOnMobileCannotSeeReportedIndicatorOnPostsReportedByAnotherUserOnPostsListPageAndCanReportThatPost() {
+    final String postId = postData.get(POST_ON_MOBILE_ON_POSTS_LIST).getId();
+
+    PageWithPosts page = new PostsListPage().open();
+
+    assertThatPostReportedByOtherUserDoesNotHaveReportedIndicatorAndCanBeReportedByCurrentUser(page, postId);
+  }
+
+  @Test(groups = "discussions-loggedInUsersMobileReporting",
+      dependsOnMethods = "userOnMobileCanReportPostOnPostDetailsPage")
+  @Execute(asUser = User.USER_2)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void userOnMobileCannotSeeReportedIndicatorOnPostsReportedByAnotherUserOnPostDetailsPageAndCanReportThatPost() {
+    final String postId = postData.get(POST_ON_MOBILE_ON_POST_DETAILS).getId();
+
+    PageWithPosts page = new PostDetailsPage().open(postId);
+
+    assertThatPostReportedByOtherUserDoesNotHaveReportedIndicatorAndCanBeReportedByCurrentUser(page, postId);
+  }
+
+  @Test(groups = "discussions-loggedInUsersMobileReporting",
+      dependsOnMethods = "userOnMobileCanReportPostOnUserPostsPage")
+  @Execute(asUser = User.USER_2)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void userOnMobileCannotSeeReportedIndicatorOnPostsReportedByAnotherUserOnUserPostsPageAndCanReportThatPost() {
+    final PostEntity.Data postData = this.postData.get(POST_ON_MOBILE_ON_USER_PAGE);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "postData" which hides the field declared at line 41. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)